docs/reclaimspace.md

```diff
@@ -101,6 +101,29 @@ data-pvc-1642664617   */1 * * * *                                   3s
 [go_cron]:
     https://pkg.go.dev/github.com/robfig/cron/v3#hdr-CRON_Expression_Format
 
+## Annotating StorageClass
+
+You can create `ReclaimSpaceCronJob` CR automatically by adding the
+`reclaimspace.csiaddons.openshift.io/schedule: "@midnight"` annotations to the
+StorageClass object. This will only inspect new PersistentVolumeClaims created from
+this StorageClass for ReclaimSpace operations. You must restart the controller if you
+want kubernetes-csi-addons to consider existing PersistentVolumeClaims.
```

Comment:
Does that mean that on a kubernetes-csi-addons controller restart, all existing PVCs from the StorageClass will get the annotation added? Maybe explain that a little more.